explorer/deploy/webAppCreator.ts

```diff
@@ -7,7 +7,6 @@ import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry';
 import { Registry } from 'azure-arm-containerregistry/lib/models';
```

Comment:
Why did NOTICE.html change?  Do you need to rebase?

Reply:
I had lots of conflicts with NOTICE.html. As to why, I wasn't sure myself. Since there's no reason for this branch to change it, I took the NOTICE.html from the latest snapshot in master and placed it here. I'll look into this again.

Reply:
Still an issue.

Reply:
Should no longer be an issue in current PR.